pallets/staking/src/benchmarking.rs

```diff
@@ -145,7 +140,7 @@ benchmarks! {
     unbond {
         let u in ...;
         let (_, controller) = create_stash_controller::<T>(u, 100)?;
-        let amount = get_minimum_balance::<T>() * 10.into();
+        let amount = T::Currency::minimum_balance() * 10.into();
```

Comment:
This was an intentional change because we have `minimum_balance()` set to 0 and that causes the benchmark test fail so to avoid it we used `get_minimum_balance::<T>` which I think to return 100 instead of 0.

Any way we can keep as it is as @maxsam4 is already working on to fix benchmarking then we can address this issue or keep the old stuff.